install-deps.sh

```diff
@@ -407,6 +392,14 @@ EOF
             $SUDO apt-get install -y /var/cache/apt/archives/{$motr_pkg,$motr_dev_pkg}
             $SUDO apt-get install -y libisal-dev
         fi
+        # install pmdk libraries
```

Comment:
Hi @tchaikov , is it here? Please have a look.

Reply:
no, please check `debian/control` to see how we manage `Build-Depends` and build profiles like `pkg.ceph.jaeger`.

Reply:
Done.